src/content/en/updates/2018/09/chrome-70-media-updates.md

```diff
@@ -0,0 +1,128 @@
+
+- [Cross-codec and cross-bytestream buffering and
+  playback](#sourcebuffer-changetype) is possible in MSE.
+- Chrome now supports [Opus in MP4 with MSE](#opus-in-mp4-for-mse).
+- Protected content playback is [allowed by
+  default](#protected-content-allowed-by-default) on Android.
+
+## Support for codec and container switching in MSE {: #sourcebuffer-changetype }
+
+Chrome is [adding support] for improved cross-codec or cross-bytestream
+transitions in [Media Source Extensions] (MSE) playback using a new
+`changeType()` method on `SourceBuffer`. It basically allows the type of media
+bytes subsequently appended to the `SourceBuffer` to be changed.
+
+While the [W3C REC version of MSE] supports adaptive playback of media, that
+adaptation requires that any media appended to a `SourceBuffer` must conform to
+the MIME type provided when initially creating the `SourceBuffer` via
+`MediaSource.addSourceBuffer(type)`, and codecs from that type and any
+previously parsed initialization segments must remain the same throughout. This
+means the website has to take explicit steps to accomplish codec or bytestream
+switching (by using multiple media elements or `SourceBuffer` tracks and
+switching among those), increasing application complexity and user-visible
+latency (such transitions require the web app to take synchronous action on the
+renderer main thread). This transition latency impairs the smoothness of media
+playback across transitions.
+
+With the new `changeType()` method on `SourceBuffer`, a `SourceBuffer` can
+buffer and support playback across different [bytestream formats] and codecs.
+This new method retains previously buffered media modulo future MSE coded frame
+eviction or removal, and leverages the splicing and buffering logic in the
+existing MSE coded frame processing algorithm.
+
+    const sourceBuffer = myMediaSource.addSourceBuffer('video/webm; codecs="opus, vp09.00.10.08"');
+    sourceBuffer.appendBuffer(someWebmOpusVP9Data);
+
+    // Later on...
+    if ('changeType' in sourceBuffer) {
+      // Change source buffer type and append new data.
+      sourceBuffer.changeType('video/mp4; codecs="mp4a.40.5, avc1.4d001e"');
+      sourceBuffer.appendBuffer(someMp4AacAvcData);
+    }
+
+As expected, if the passed type is not supported by the browser, this method
+throws an error.
+
+Check out the [sample] to play with cross-codec and
+cross-bytestream buffering and playback of an audio element.
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/K_OFPxA_whE) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5719220952236032) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=605134)
+
+## Opus in MP4 for MSE {: #opus-in-mp4-for-mse }
+
+The open and highly versatile audio codec [Opus] has been supported in regular
+Chrome media playback since Chrome 33. [Opus in ISO-BMFF] support (aka Opus in
```

Comment:
Say 'in the `<video>` element' instead of 'in regular Chrome media playback'. 

My issue is that 'regular' is so vague it didn't immediately register that you were talking about something other than MSE. So the next two sentences sounded redundant and I was confused.

Reply:
Done.
I've added `<audio>` as well.